### ISSUE TITLE
Add `UNSET TBLPROPERTIES`

### ISSUE THREAD

**Issue by ion-elgreco** [delta-io/delta-rs#2073]

**Comment by abhiaagarwal** [comment]
Will this be related to the "alter" namespace proposed in #1909? I need to implement a few of them for an application I'm developing

**Comment by ion-elgreco** [comment]
@abhiaagarwal yup, already have a PR open for this

**Comment by abhiaagarwal** [comment]
@ion-elgreco  Yeah, I read through it, but it doesn't explicitly namespace everything to "alter", on either the Python or the Rust side. Is that still planned? I want to develop Alter Column, I want to know what's the preferred format for developing a something in `core/operations'

**Comment by abhiaagarwal** [comment]
Sorry, I was unclear — I see that it does exist in Python, but will the idea of the "alter namespace" be exported to rust as well? I'm using delta-rs primarily in rust. I'm wondering how to group all Alter-related expressions together on the rust side to be exported into the alter namespace on the python, or to just handle it on the python side exclusively.

**Comment by ion-elgreco** [comment]
You don't really need to group them in rust, the grouping is mainly done in python under TableAlterer

**Comment by ion-elgreco** [comment]
You don't really need to group them in rust, the grouping is mainly done in python under TableAlterer, you just expose it operation as a method under RawDeltaTable
